src/components/mainpage.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import styled from 'styled-components';
+import { v4 as uuidv4 } from 'uuid';
+import CircleIcon from '@mui/icons-material/Circle';
+import FacebookRoundedIcon from '@mui/icons-material/FacebookRounded';
+import InstagramIcon from '@mui/icons-material/Instagram';
+import TwitterIcon from '@mui/icons-material/Twitter';
+import { Splide, SplideSlide } from '@splidejs/react-splide';
+import '@splidejs/react-splide/css';
+import yatches from './dummydata/joydata';
+import './mainpage.css';
+
+const MainPage = () => (
+  <div>
+    <Wrapper>
+      <h2>Popular Destinations</h2>
+      <h3> Select your desired destination</h3>
+      <div className="dots">
+        {[...Array(15)].map(() => (
+          <div key={uuidv4()}>
+            <CircleIcon color="disabled" sx={{ fontSize: '0.5rem' }} />
+          </div>
+        ))}
+      </div>
+      <Splide
+        options={{
+          perPage: 3,
+          arrows: true,
+          pagination: false,
+          drag: 'free',
+          gap: '1rem',
+        }}
+      >
+        {yatches.map((item) => (
+          <SplideSlide key={item.id}>
+            <Card>
+              <div className="card">
+                <img src={item.image} alt={item.name} />
+                <div className="yatch_name">
+                  <h4>{item.name}</h4>
+                </div>
+                <div className="dots">
+                  {[...Array(15)].map(() => (
+                    <div key={uuidv4()}>
+                      <CircleIcon
+                        color="disabled"
+                        sx={{ fontSize: '0.7rem' }}
+                      />
+                    </div>
+                  ))}
+                </div>
+                <p>{item.description}</p>
+                <div className="social">
+                  <FacebookRoundedIcon
+                    sx={{
+                      fontSize: '1.5rem',
+                      color: 'rgb(0, 0, 0, 0.26)',
+                      marginRight: '1rem',
+                    }}
+                  />
+                  <InstagramIcon
```

Comment:
How about creating a yatch_card component to put the details of a yatch that you will then render in the main page. Then we will do the same thing for the side navbar and other components to keep each file as specific as possible.